Neos.Neos/Classes/Service/NodeTypeSchemaBuilder.php

```diff
@@ -54,19 +63,33 @@ public function generateNodeTypeSchema()
             'constraints' => $this->generateConstraints()
         ];
 
-        $nodeTypes = $this->nodeTypeManager->getNodeTypes(true);
-        /** @var NodeType $nodeType */
+        // We need skip abstract nodetypes as they are not instantiated by the UI
+        $nodeTypes = $this->nodeTypeManager->getNodeTypes(false);
+
+        // Get special abstract nodetypes required by the Neos.UI
+        if ($this->nodeTypeRoles) {
+            $additionalNodeTypeNames = array_values($this->nodeTypeRoles);
+            foreach ($additionalNodeTypeNames as $additionalNodeTypeName) {
+                $nodeTypes[$additionalNodeTypeName] = $this->nodeTypeManager->getNodeType($additionalNodeTypeName);
+            }
+        }
+
         foreach ($nodeTypes as $nodeTypeName => $nodeType) {
             if ($nodeType->isAbstract() === false) {
                 $configuration = $nodeType->getFullConfiguration();
                 $schema['nodeTypes'][$nodeTypeName] = $configuration;
                 $schema['nodeTypes'][$nodeTypeName]['label'] = $nodeType->getLabel();
+
+                // Remove the postprocessors, as they are not needed in the UI
+                unset($schema['nodeTypes'][$nodeTypeName]['postprocessors']);
```

Comment:
Why is that done only for non-abstract types?

Reply:
Hm, could be done for all. But we only have 4 abstract types from the UI `nodeTypeRoles`. I can move the line!